From 7f80c74569932b0db74d0ca5072fbce461e54e76 Mon Sep 17 00:00:00 2001
From: Henry Lau <henry.lau@intel.com>
Date: Thu, 21 Feb 2013 21:11:35 -0800
Subject: [PORT FROM MAIN] Remove all debug symbols from user builds

BZ: 87067

Remove all debug symbols from user builds for 2 reasons:
- security concerns
- reduce library file size by 30%

Change-Id: Ia1ac08894ca5360322d696b15029e2697d93123a
Orig-Change-Id: I7ddb47ee3aba3fe271c95e8932f0619091de0319
Reviewed-on: http://android.intel.com:8080/93191
Reviewed-by: Yin, Fengwei <fengwei.yin@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Imberton, Guilhem <guilhem.imberton@intel.com>
Tested-by: Imberton, Guilhem <guilhem.imberton@intel.com>
Reviewed-by: Beare, Bruce J <bruce.j.beare@intel.com>
Reviewed-by: Guerard, Christophe <christophe.guerard@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 core/combo/TARGET_linux-x86.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/core/combo/TARGET_linux-x86.mk b/core/combo/TARGET_linux-x86.mk
index ebb6257..ed88701 100644
--- a/core/combo/TARGET_linux-x86.mk
+++ b/core/combo/TARGET_linux-x86.mk
@@ -56,7 +56,7 @@ TARGET_LD := $(TARGET_TOOLS_PREFIX)ld$(HOST_EXECUTABLE_SUFFIX)
 TARGET_STRIP := $(TARGET_TOOLS_PREFIX)strip$(HOST_EXECUTABLE_SUFFIX)
 
 ifeq ($(TARGET_BUILD_VARIANT),user)
-TARGET_STRIP_COMMAND = $(TARGET_STRIP) --strip-debug $< -o $@
+TARGET_STRIP_COMMAND = $(TARGET_STRIP) --strip-all $< -o $@
 else
 TARGET_STRIP_COMMAND = $(TARGET_STRIP) --strip-debug $< -o $@ && \
 	$(TARGET_OBJCOPY) --add-gnu-debuglink=$< $@
-- 
1.7.4.1

